package.json

```diff
@@ -18,8 +18,8 @@
   "author": "Relaying Services Team",
   "license": "MIT",
   "dependencies": {
-    "@rsksmart/rif-relay-common": "https://github.com/infuy/rif-relay-common#PP-94/revenue-sharing-model",
-    "@rsksmart/rif-relay-contracts": "https://github.com/anarancio/rif-relay-contracts#PP-94/revenue-sharing-model",
+    "@rsksmart/rif-relay-common": "https://github.com/antomor/rif-relay-common#PP-94/mandatory-collector-address",
+    "@rsksmart/rif-relay-contracts": "https://github.com/antomor/rif-relay-contracts#PP-94/fix-truffle-typings-patch",
```

Comment:
We need to revert this change after merging the PRs:
- [rif-relay-common](https://github.com/infuy/rif-relay-common/pull/9)
- [rif-relay-contracts](https://github.com/anarancio/rif-relay-contracts/pull/21)